ci/rpm7/build-dep.sh

```diff
@@ -44,24 +45,39 @@ yum -y install \
   tar \
   cyrus-sasl-devel
 
-curl -L https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz | tar xzC /opt/ \
+# FIXME: remove legacy driver when it gets discontinued in the code
+echo "INSTALL: mongodb legacy c++ driver" \
+&& curl -L https://github.com/mongodb/mongo-cxx-driver/archive/legacy-1.1.2.tar.gz | tar xzC /opt/ \
 && cd /opt/mongo-cxx-driver-legacy-1.1.2 \
 && scons --disable-warnings-as-errors --use-sasl-client --ssl \
 && scons install --disable-warnings-as-errors --prefix=/usr/local --use-sasl-client --ssl \
 && rm -Rf /opt/mongo-cxx-driver-legacy-1.1.2
 
-curl -L https://github.com/miloyip/rapidjson/archive/v1.1.0.tar.gz | tar xzC /opt/ \
+echo "INSTALL: mongodb c driver (required by mongo c++ driver)" \
+&& curl -L https://github.com/mongodb/mongo-c-driver/releases/download/1.16.0/mongo-c-driver-1.16.0.tar.gz | tar xzC /opt/ \
+&& cd /opt/mongo-c-driver-1.16.0 \
+&& mkdir cmake-build \
+&& cd cmake-build \
+&& cmake3 -DENABLE_AUTOMATIC_INIT_AND_CLEANUP=OFF .. \
```

Comment:
NO or OFF?

Check: http://mongoc.org/libmongoc/current/init-cleanup.html#deprecated-feature-automatic-initialization-and-cleanup

Reply:
OFF is fine. See https://jira.mongodb.org/browse/CDRIVER-3903